backend/types.py

```diff
@@ -139,37 +146,23 @@ class EmbedderConfig(ModelConfig):
     Embedder configuration
     """
 
-    # This field will probably be removed soon or refactored
-    embedding_model_config: ModelConfig = Field(
-        alias="model_config",
-    )
-    config: Optional[Dict[str, Any]] = Field(
-        title="Configuration for the embedder", default_factory=dict
-    )
-
-    @model_serializer
-    def serialize(self):
-        return {
-            "model_config": self.embedding_model_config,
-            "config": self.config,
-        }
+    pass
 
 
 class ParserConfig(ModelConfig):
     """
     Parser configuration
     """
 
-    chunk_size: int = Field(title="Chunk Size for data parsing", ge=1, default=1000)
-    chunk_overlap: int = Field(title="Chunk Overlap for indexing", ge=0, default=20)
-    parser_map: Dict[str, str] = Field(
-        title="Mapping of file extensions to parsers",
-        default_factory=dict,
-    )
-    additional_config: Optional[Dict[str, Any]] = Field(
-        title="Additional optional configuration for the parser",
-        default_factory=dict,
-    )
+    type: ModelType = ModelType.parser
```

Comment:
We probably need use_enum_values equivalent behavior to serialize correctly, check once
We need to check all models where we use enums

Reply:
done